src/JitInterface/src/PrimitiveTypeDescriptor.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+using System.Diagnostics;
+
+namespace Internal.JitInterface
+{
+    public class PrimitiveTypeDescriptor
+    {
+        internal enum TYPE_ENUM
```

Comment:
I called it Internal.JitInterface.TypesDebugInfo because someday it will write DWARF too.